### PR TITLE
Improve CI.

### PR DESCRIPTION
**Pull request by felixfontein**
Fix some bugs in the GHA workflow.

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @gundalow

**Comment by felixfontein** [comment]
(CI is expected to fail.)

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -99,7 +99,9 @@ jobs:
       - name: Set up Python ${{ matrix.ansible }}
```
Comment (webknjaz) [inline .github/workflows/ansible-test.yml:99]: @felixfontein this also uses the wrong ansible version var

File: .github/workflows/ansible-test.yml [hunk 2]
```diff
@@ -165,7 +167,9 @@ jobs:
...
```
Comment (webknjaz) [inline .github/workflows/ansible-test.yml:167]: also wrong var

File: .github/workflows/ansible-test.yml (hunk 1 above)
Comment (felixfontein) [reply .github/workflows/ansible-test.yml:99]: See #19.

**Pull request merged**